### DISCUSSION TITLE
Valijson and JSON Schema

### DISCUSSION THREAD

**Discussion by jviotti** [microsoft/winget-cli#4728] [General]
Hey there! I'm a TSC member of JSON Schema focused on building better and faster JSON Schema utilities for C++.

I saw this repo makes use of Valijson and its (limited) JSON Schema implementation. How is your use of Valijson going? Anything you like or dislike? Did you find a need for supporting newer dialects of JSON Schema (such as 2020-12)?

Any honest feedback would be super appreciated to make JSON Schema and my C++ efforts in there more valuable! 🙏🏻

**Comment by davidldennison** [comment]
Hi!

I have a few things I could put together that I've had a need for, but it's more in relation to the Winget-CLI configuration file and its use of JSON. I also do SEO and am trying to build an SEO tool utilizing JSON Schema, but I'm not sure if either of these are in the scope of what you're looking for. If it is, I can definitely get together some feedback!

**Reply by jviotti** [reply]
Hi @davidldennison , please do! 🙏🏻 Any feedback so we can better direct our efforts to solving developer's problems is very appreciated

**Comment by denelon** [comment]
@jviotti maybe @JohnMcPMS has some input.
